pyvrp/cpp/Individual.h

```diff
@@ -3,6 +3,7 @@
 
 #include "Measure.h"
 #include "ProblemData.h"
+#include "TimeWindowSegment.h"
```

Comment:
It's one thing that we use the concatenation scheme in the implementation, but I'd really like to avoid it in the declarations. In the internal state of `Individual::Route`, can we store only what we need, rather than the whole `TWS`? Then this include here can be avoided.

Reply:
With the same reasoning as before I don't really see the problem, I think it's nice we can group the time related attributes where we need all of them (duration, time warp, earliest and latest) except the indices in TWS (which I'd rather get rid of as well). This object just holds timing info for the complete route, which is what the TWS is intended for (+ some logic related to it). If you feel so strongly I can change it but I don't see the point of reimplementing mostly the same attributes rather than holding the struct? Or is there some technical aspect I'm missing?

Reply:
@N-Wouda is this a problem or can we keep it?